met-web/public/index.html

```diff
@@ -3,46 +3,23 @@
   <head>
     <script src="%PUBLIC_URL%/config/config.js"></script>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.png" />
```

Comment:
.png for favicon? Wouldn't it be heavy for it. We can generate a .ico for png here https://favicon.io/favicon-converter/

Reply:
They used png for the bcgov repo